cmake/modules/FindPipewire.cmake

```diff
@@ -12,7 +12,7 @@
 #
 
 if(PKG_CONFIG_FOUND)
-  pkg_check_modules(PC_PIPEWIRE libpipewire-0.3>=0.3.24 QUIET)
+  pkg_check_modules(PC_PIPEWIRE libpipewire-0.3>=0.3.34 QUIET)
```

Comment:
Even Ubuntu 22.04 has 0.3.48 already. From support POV, I would choose a version that can be supported at least by us. Think not only about PT issues, but also lots of stuff that has changed for delay management just recently. If we have proper support in V21 only, it will also only hit ubuntu + 2 from now.

Reply:
I'm fine bumping it up to [0.3.48](https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/0.3.48). The progress with pipewire is moving fast and the latest release is already [0.3.64 ](https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/0.3.64) with [0.3.48](https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/0.3.48) shipping only ~10 months ago.